src/black/linegen.py

```diff
@@ -834,7 +834,10 @@ def left_hand_split(
                 current_leaves = tail_leaves if body_leaves else head_leaves
             current_leaves.append(leaf)
             if current_leaves is head_leaves:
-                if leaf.type == leaf_type:
+                if leaf.type == leaf_type and (
+                    Preview.fix_type_expansion_split
+                    and not (leaf_type == token.LPAR and depth > 0)
```

Comment:
```suggestion
                    Preview.fix_type_expansion_split not in mode
                    or not (leaf_type == token.LPAR and depth > 0)
```
That should be it, looks like the function already takes `mode` as a parameter, so we can just use it.